.github/actions/build-manager-image/action.yaml

```diff
@@ -24,16 +24,39 @@ runs:
         registry: europe-central2-docker.pkg.dev
         username: _json_key_base64
         password: ${{ inputs.push-sa-key }}
-    - name: Build and Push
+    - name: Build and Push linux/amd64
       if: ${{ inputs.push-image == 'true' }}
       uses: docker/build-push-action@v6
       with:
         context: .
         platforms: 'linux/amd64'
+        provenance: false
         push: true
-        tags: ${{ inputs.operator-image-name }}
+        tags: ${{ inputs.operator-image-name }}-amd64
+        cache-from: type=gha
+        cache-to: type=gha,mode=max
+    - name: Build and Push linux/arm64
+      if: ${{ inputs.push-image == 'true' }}
+      uses: docker/build-push-action@v6
+      with:
+        context: .
+        platforms: 'linux/arm64'
```

Comment:
Why do you separate the builds and tag both images separately? That's not how Docker [documentation](https://docs.docker.com/build/ci/github-actions/multi-platform/) recommends. You should extend `platforms` with all platforms you want for the images to be put under the same image manifest. Then Docker/CRI automatically downloads image for given architecture.
```
platforms: 'linux/amd64, linux/arm64'`
```